### PR TITLE
EOS-15537 Configuring Stonith on N>2 servers

### PR DESCRIPTION
**Pull request by shindeam**
POC for stonith on > 3 node HW cluster 
- Self fencing using Shared stonith device between nodes (Did POC using iscsi server on VMs.)
- Opposite fencing using stonith device created for one node & will be responsible for doing action on other nodes. If anything wrong happened on the nodes and goes offline in cluster stonith device will take action to make that node power off (Did POC using BMC ipmi nodes and 3 node hw)

### CODE REVIEW

File: Stonith/docs/3nodes/scsi_stonith.md [hunk 1]
```diff
@@ -0,0 +1,683 @@
+### Setup on 3 VM's
+
```
Comment (ajay-paratmandali) [inline Stonith/docs/3nodes/scsi_stonith.md:2]: Can you separate scsi and ipmi
and add in https://github.com/Seagate/cortx-experiments/tree/main/Stonith/docs
depend on scsi and ipmi

File: Stonith/docs/3nodes/scsi_stonith.md [hunk 2]
```diff
@@ -0,0 +1,683 @@
...
+1. Did pcs cluster setup on below 3 nodes by following the documents:
```
Comment (ajay-paratmandali) [inline Stonith/docs/3nodes/scsi_stonith.md:3]: Can you separate scsi and ipmi
and add in https://github.com/Seagate/cortx-experiments/tree/main/Stonith/docs
depend on scsi and ipmi

**Review by ajay-paratmandali — CHANGES_REQUESTED** [review]

File: Stonith/docs/3nodes/scsi_stonith.md (hunk 2 above)
Comment (shindeam) [reply Stonith/docs/3nodes/scsi_stonith.md:3]: Yeahh agree, Separated out them in a different files.

Comment (ajay-paratmandali) [reply Stonith/docs/3nodes/scsi_stonith.md:3]: can we move in
Stonith/docs/scsi/3node_scsi_stonith.md
Stonith/docs/ipmi/3node_ipmi_stonith.md
scsi and ipmi are already present

File: Stonith/docs/3nodes/scsi_stonith.md [hunk 3]
```diff
@@ -0,0 +1,504 @@
...
+ Current DC: node2 (version 1.1.23-1.el7-9acf116022) - partition with quorum
+ Last updated: Thu Jan  7 23:50:23 2021
+ Last change: Thu Jan  7 03:01:36 2021 by root via cibadmin on node1
+ 3 nodes configured
+ 1 resource instance configured
+
+PCSD Status:
+  node2: Online
+  node1: Online
+  node3: Online
```
Comment (ajay-paratmandali) [inline Stonith/docs/3nodes/scsi_stonith.md:503]: Add result 
is it pass or failed
like
https://github.com/Seagate/cortx-experiments/blob/main/Stonith/docs/scsi/scsi_stonith.md

File: Stonith/docs/3nodes/ipmi_stonith.md [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+ Clone Set: stonith-c3-clone [stonith-c3]
+     Started: [ node1 node2 ]
+     Stopped: [ node3 ]
+
+Daemon Status:
+  corosync: active/enabled
+  pacemaker: active/enabled
+  pcsd: active/enabled
+
+Means node3 is stopped and we are good with our POC.
```
Comment (ajaysrivas) [inline Stonith/docs/3nodes/ipmi_stonith.md:140]: So node3 was not reachable by node1 and node2.
Did fencing was attempted by both the nodes, node1 and node2?
Let's node1 was able to fence node3. What will be the status of stonith-c3-clone on node2 trying to fence node1?

File: Stonith/docs/3nodes/ipmi_stonith.md [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+     Started: [ node1 node2 ]
+     Stopped: [ node3 ]
+
+Daemon Status:
+  corosync: active/enabled
+  pacemaker: active/enabled
+  pcsd: active/enabled
+[root@smc40-m09 cluster]# 
+
+```
```
Comment (ajay-paratmandali) [inline Stonith/docs/3nodes/ipmi_stonith.md:178]: Add final result

File: Stonith/docs/3nodes/scsi_stonith.md (hunk 2 above)
Comment (shindeam) [reply Stonith/docs/3nodes/scsi_stonith.md]: Done

File: Stonith/docs/3nodes/scsi_stonith.md (hunk 3 above)
Comment (shindeam) [reply Stonith/docs/3nodes/scsi_stonith.md]: For VM's testing I haven't taken logs so for now I just added the steps of result. For node hw I have taken out the logs for each stage.

File: Stonith/docs/3nodes/ipmi_stonith.md (hunk 4 above)
Comment (shindeam) [reply Stonith/docs/3nodes/ipmi_stonith.md]: @ajaysrivas I have added minute details in result section in the file.

File: Stonith/docs/3nodes/ipmi_stonith.md (hunk 5 above)
Comment (shindeam) [reply Stonith/docs/3nodes/ipmi_stonith.md]: Added, Done.

**Review by ajay-paratmandali — APPROVED** [review]

File: Stonith/docs/3nodes/ipmi_stonith.md (hunk 4 above)
Comment (ajaysrivas) [reply Stonith/docs/3nodes/ipmi_stonith.md]: @shindeam The logs are included in the result but actual question is not answered. Anyone looking at the results, needs to analyze logs. You should conclude the behavior.
Also, where is the measure of n/w trafiic?

Comment (shindeam) [reply Stonith/docs/3nodes/ipmi_stonith.md]: Ok agree, added result in words.

Comment (ajaysrivas) [reply Stonith/docs/3nodes/ipmi_stonith.md]: Please check this PR - https://github.com/Seagate/cortx-experiments/pull/8/files
nload tool was used by Andrei for measuring n/w traffic.

Comment (shindeam) [reply Stonith/docs/3nodes/ipmi_stonith.md]: yes used same & added the network measures for all nodes. updated in document & ticket comment too

**Pull request merged**